From c25c027f4dc0de7f248d35f08718533e1f688a33 Mon Sep 17 00:00:00 2001
From: BJ DevOps Team <bjdevops@NXP1.onmicrosoft.com>
Date: Thu, 1 Nov 2018 11:10:30 +0800
Subject: [PATCH] Makefile: add -Wno-misleading-indentation and
 -Wno-stringop-truncationto cflags

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f0755f2..6cb077c 100644
--- a/Makefile
+++ b/Makefile
@@ -419,7 +419,7 @@ KBUILD_CFLAGS   := -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs \
 		   -fno-strict-aliasing -fno-common -fshort-wchar \
                    -Wno-packed-not-aligned \
                    -Wno-tautological-compare \
-		   -Werror-implicit-function-declaration \
+		   -Werror-implicit-function-declaration -Wno-stringop-truncation -Wno-misleading-indentation\
 		   -Wno-format-security \
 		   -std=gnu89
 KBUILD_CPPFLAGS := -D__KERNEL__
-- 
1.8.3.1

