From 4739dd692eadb1ba4477bb386f05ee963d076c7a Mon Sep 17 00:00:00 2001
From: Zongchun Yu <zongchun.yu@nxp.com>
Date: Tue, 23 Jun 2020 12:51:21 +0800
Subject: [PATCH] scripts/sign_encrypt.py: Correct the Crypto module name

Signed-off-by: Zongchun Yu <zongchun.yu@nxp.com>
---
 scripts/sign_encrypt.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/scripts/sign_encrypt.py b/scripts/sign_encrypt.py
index ffca483a..a4b13a9c 100755
--- a/scripts/sign_encrypt.py
+++ b/scripts/sign_encrypt.py
@@ -128,9 +128,9 @@ def get_args(logger):
 
 
 def main():
-    from Cryptodome.Signature import pss
-    from Cryptodome.Hash import SHA256
-    from Cryptodome.PublicKey import RSA
+    from Crypto.Signature import pss
+    from Crypto.Hash import SHA256
+    from Crypto.PublicKey import RSA
     import base64
     import logging
     import os
@@ -169,7 +169,7 @@ def main():
     shdr_version = struct.pack('<I', hdr_version)
 
     if args.enc_key:
-        from Cryptodome.Cipher import AES
+        from Crypto.Cipher import AES
         cipher = AES.new(bytearray.fromhex(args.enc_key), AES.MODE_GCM)
         ciphertext, tag = cipher.encrypt_and_digest(img)
 
-- 
2.17.1

