Upstream-Status: Inappropriate [embedded-specific]

Signed-off-by: Chunrong Guo <chunrong.guo@nxp.com>

--- a/src/import/hack/lib/golang.shold	2017-06-30 06:53:16.000000000 +0800
+++ b/src/import/hack/lib/golang.sh	2018-05-02 14:32:40.706915559 +0800
@@ -668,9 +668,11 @@
         parallel=false
       fi
     fi
-
+    local goarch_backup=${GOARCH}
+    export GOARCH=$(go env GOHOSTARCH)
     # First build the toolchain before building any other targets
     kube::golang::build_kube_toolchain
+    export GOARCH="$goarch_backup"
 
     kube::log::status "Generating bindata:" "${KUBE_BINDATAS[@]}"
     for bindata in ${KUBE_BINDATAS[@]}; do
