From b08dfc7c1bb61b5be3c1818204ab6f3d095dc7fa Mon Sep 17 00:00:00 2001
From: Chunrong Guo <chunrong.guo@nxp.com>
Date: Tue, 20 Nov 2018 17:27:48 +0800
Subject: [PATCH] remove  Obsolete configurations

Signed-off-by: Chunrong Guo <chunrong.guo@nxp.com>
---
 gcc/config.gcc | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/gcc/config.gcc b/gcc/config.gcc
index 202ac2a..bca8c30 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -235,18 +235,18 @@ xm_file=
 md_file=
 
 # Obsolete configurations.
-case ${target} in
-  powerpc*-*-*spe*			\
-  | tile*-*-*				\
- )
-    if test "x$enable_obsolete" != xyes; then
-      echo "*** Configuration ${target} is obsolete." >&2
-      echo "*** Specify --enable-obsolete to build it anyway." >&2
-      echo "*** Support will be REMOVED in the next major release of GCC," >&2
-      echo "*** unless a maintainer comes forward." >&2
-      exit 1
-    fi;;
-esac
+#case ${target} in
+#  powerpc*-*-*spe*			\
+#  | tile*-*-*				\
+# )
+#    if test "x$enable_obsolete" != xyes; then
+#      echo "*** Configuration ${target} is obsolete." >&2
+#      echo "*** Specify --enable-obsolete to build it anyway." >&2
+#      echo "*** Support will be REMOVED in the next major release of GCC," >&2
+#      echo "*** unless a maintainer comes forward." >&2
+#      exit 1
+#    fi;;
+#esac
 
 # Unsupported targets list.  Do not put an entry in this list unless
 # it would otherwise be caught by a more permissive pattern.  The list
-- 
2.7.4

