From 273a848610d9e33a9edf7527d6ac4c562eb1ce45 Mon Sep 17 00:00:00 2001
From: Chunrong Guo <chunrong.guo@nxp.com>
Date: Thu, 26 Jul 2018 15:52:02 +0800
Subject: [PATCH] Makefile: remove openssl

Signed-off-by: BJ DevOps Team <bjdevops@NXP1.onmicrosoft.com>
---
 Makefile | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index 8925cf1..595cab7 100644
--- a/Makefile
+++ b/Makefile
@@ -29,7 +29,7 @@ CGO_LDFLAGS += -L${OPENSSL_PATH}
 CGO_CFLAGS += -I${OPENSSL_PATH}/include
 
 
-kubelet:goenv fetch-kube kube-openssl
+kubelet:goenv fetch-kube
 	export GOROOT=$(GOROOT) && \
 	export GOPATH=$(GOPATH) && \
 	export PATH=$(GOROOT)/bin:$(PATH) && \
@@ -68,13 +68,5 @@ fetch-kube:
 		mv kubernetes-$(KUBE_VERSION) $(GOPATH)/src/k8s.io/kubernetes; \
 	fi
 
-kube-openssl:
-	rm -rf $(GOPATH)/src/openssl;cp -rf $(GO_OPENSSL_PATH) $(GOPATH)/src/openssl; \
 
-	sed -i '/"openssl"/d' $(GOPATH)/src/k8s.io/kubernetes/staging/src/k8s.io/client-go/transport/transport.go; \
-	sed -i '/net\/http/a\\t"openssl"' $(GOPATH)/src/k8s.io/kubernetes/staging/src/k8s.io/client-go/transport/transport.go; \
-
-	sed  -i 's/tls.X509KeyPair(c.TLS.CertData, c.TLS.KeyData)/openssl.X509KeyPair(c.TLS.CertData, c.TLS.KeyData, c.TLS.KeyFile)/g' \
-	$(GOPATH)/src/k8s.io/kubernetes/staging/src/k8s.io/client-go/transport/transport.go; \
-
-.PHONY: kubelet clean install goenv fetch-kube kube-openssl
+.PHONY: kubelet clean install goenv fetch-kube
-- 
1.8.3.1

