From ce7d006f7d962c9e1777392a98eed2f3984365c5 Mon Sep 17 00:00:00 2001
From: Chunrong Guo <chunrong.guo@nxp.com>
Date: Wed, 9 Jan 2019 10:11:49 +0100
Subject: [PATCH] support aarch64_be

Signed-off-by: C.r. Guo <nxa13725@lsv07005.swis.us-cdc01.nxp.com>
---
 mesonbuild/environment.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mesonbuild/environment.py b/mesonbuild/environment.py
index b4fc0dc..014ecab 100644
--- a/mesonbuild/environment.py
+++ b/mesonbuild/environment.py
@@ -74,6 +74,7 @@ build_filename = 'meson.build'
 known_cpu_families = (
     'allarch',
     'aarch64',
+    'aarch64_be',
     'arc',
     'arm',
     'e2k',
-- 
2.7.4

