From 6b7b78ff8355036946683c58f283d672bf0f0ad2 Mon Sep 17 00:00:00 2001
From: BJ DevOps Team <bjdevops@NXP1.onmicrosoft.com>
Date: Fri, 12 Oct 2018 10:01:04 +0800
Subject: [PATCH] support powerpc64

---
 mesonbuild/environment.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mesonbuild/environment.py b/mesonbuild/environment.py
index 44e3fb0..5cc2275 100644
--- a/mesonbuild/environment.py
+++ b/mesonbuild/environment.py
@@ -85,6 +85,7 @@ known_cpu_families = (
     'parisc',
     'ppc',
     'ppc64',
+    'powerpc64',
     'riscv32',
     'riscv64',
     's390x',
-- 
1.8.3.1

