From 163a2a8e469c80b9069076f9c9f941c2cdae2219 Mon Sep 17 00:00:00 2001
From: Chunrong Guo <chunrong.guo@nxp.com>
Date: Fri, 12 Oct 2018 11:32:46 +0800
Subject: [PATCH] support aarch64_be

Signed-off-by: BJ DevOps Team <bjdevops@NXP1.onmicrosoft.com>
---
 mesonbuild/environment.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mesonbuild/environment.py b/mesonbuild/environment.py
index 5cc2275..4d7aae7 100644
--- a/mesonbuild/environment.py
+++ b/mesonbuild/environment.py
@@ -77,6 +77,7 @@ build_filename = 'meson.build'
 known_cpu_families = (
     'allarch',
     'aarch64',
+    'aarch64_be',
     'arm',
     'e2k',
     'ia64',
-- 
1.8.3.1

