--- nss-3.35.orig/nss/lib/freebl/Makefileold	2018-04-10 12:30:52.002060581 +0800
+++ nss-3.35.orig/nss/lib/freebl/Makefile	2018-04-10 11:49:02.490147469 +0800
@@ -533,7 +533,7 @@
             EXTRA_SRCS += chacha20_vec.c
         endif
     else
-        ifeq ($(CPU_ARCH),aarch64)
+        ifeq ($(CPU_ARCH),aarch64_be)
             EXTRA_SRCS += Hacl_Poly1305_64.c
         else
             EXTRA_SRCS += poly1305.c
