From 2e542522452f98fbd85fb0fadeb6411437c2d1f0 Mon Sep 17 00:00:00 2001
From: Mihai Lindner <mihai.lindner@nxp.com>
Date: Wed, 18 Sep 2019 12:16:21 +0300
Subject: [PATCH] Temp fix for conflicting types for 'getdents64'

Upstream-Status: Inappropriate [other - temporary build fix]
Signed-off-by: Mihai Lindner <mihai.lindner@nxp.com>
---
 testcases/kernel/syscalls/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/testcases/kernel/syscalls/Makefile b/testcases/kernel/syscalls/Makefile
index d1becd054..9415cc4b8 100644
--- a/testcases/kernel/syscalls/Makefile
+++ b/testcases/kernel/syscalls/Makefile
@@ -24,6 +24,8 @@ top_srcdir		?= ../../..
 
 include	$(top_srcdir)/include/mk/env_pre.mk
 
+FILTER_OUT_DIRS	+= getdents
+
 ifeq ($(UCLINUX),1)
 FILTER_OUT_DIRS	+= capget capset chmod chown clone fork getcontext llseek \
 		   mincore mprotect nftw profil remap_file_pages sbrk
-- 
2.17.1

