From c237ac32d4f1bd3bb0d0a38146032fac3b7b53a2 Mon Sep 17 00:00:00 2001
From: Chunrong Guo <chunrong.guo@nxp.com>
Date: Tue, 16 Oct 2018 15:40:54 +0800
Subject: [PATCH] Makefile:add -Wno-redundant-decls

Signed-off-by: BJ DevOps Team <bjdevops@NXP1.onmicrosoft.com>
---
 fsl_opt_lib/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index cad7792..c52e598 100644
--- a/Makefile
+++ b/Makefile
@@ -78,11 +78,11 @@ WARNING_OPTS_C = -Wall \
 		-Wswitch-default \
 		-Wmissing-declarations \
 		-Wconversion \
-		-Wredundant-decls \
+		-Wno-redundant-decls \
 		-Winline \
 		-Werror
 
-CFLAGS = -c -fpic -pipe $(CC_M_FLAG) $(CC_CPU_FLAGS) $(AS_CPU_FLAGS) $(WARNING_OPTS_C)
+CFLAGS = -c -fpic -fno-stack-protector -pipe $(CC_M_FLAG) $(CC_CPU_FLAGS) $(AS_CPU_FLAGS) $(WARNING_OPTS_C)
 LD_FLAGS += -shared -nodefaultlibs -nostdlib
 
 INCLUDES= -Iarch/$(ARCH) -Iinclude
-- 
2.7.4

