diff -Naur binutils-2.25/bfd/config.bfd binutils-2.25_new/bfd/config.bfd
--- binutils-2.25/bfd/config.bfd	2014-10-14 02:32:02.000000000 -0500
+++ binutils-2.25_new/bfd/config.bfd	2015-02-10 05:03:54.372951932 -0600
@@ -1262,7 +1262,7 @@
     targ_selvecs="rs6000_xcoff_vec powerpc_elf32_vec powerpc_elf32_le_vec powerpc_boot_vec"
     targ64_selvecs="powerpc_elf64_vec powerpc_elf64_le_vec powerpc_elf64_fbsd_vec"
     ;;
-  powerpc-*-*bsd* | powerpc-*-elf* | powerpc-*-sysv4* | powerpc-*-eabi* | \
+  powerpc-*-*bsd* | powerpc-*-elf* | powerpc-*-sysv4* | powerpc-*-eabi* | powerpc-*-aeabi* | \
   powerpc-*-solaris2* | powerpc-*-linux-* | powerpc-*-rtems* | \
   powerpc-*-chorus*)
     targ_defvec=powerpc_elf32_vec
diff -Naur binutils-2.25/config.sub binutils-2.25_new/config.sub
--- binutils-2.25/config.sub	2014-10-14 02:32:02.000000000 -0500
+++ binutils-2.25_new/config.sub	2015-02-10 05:06:05.878951927 -0600
@@ -1372,7 +1372,7 @@
 	      | -ekkobsd* | -kfreebsd* | -freebsd* | -riscix* | -lynxos* \
 	      | -bosx* | -nextstep* | -cxux* | -aout* | -elf* | -oabi* \
 	      | -ptx* | -coff* | -ecoff* | -winnt* | -domain* | -vsta* \
-	      | -udi* | -eabi* | -lites* | -ieee* | -go32* | -aux* \
+	      | -udi* | -eabi* | -aeabi* | -lites* | -ieee* | -go32* | -aux* \
 	      | -chorusos* | -chorusrdb* | -cegcc* \
 	      | -cygwin* | -msys* | -pe* | -psos* | -moss* | -proelf* | -rtems* \
 	      | -mingw32* | -mingw64* | -linux-gnu* | -linux-android* \
diff -Naur binutils-2.25/gas/configure.tgt binutils-2.25_new/gas/configure.tgt
--- binutils-2.25/gas/configure.tgt	2014-10-14 02:32:03.000000000 -0500
+++ binutils-2.25_new/gas/configure.tgt	2015-02-10 05:08:37.073952182 -0600
@@ -376,6 +376,7 @@
   ppc-*-beos*)				fmt=coff ;;
   ppc-*-*n*bsd* | ppc-*-elf*)		fmt=elf ;;
   ppc-*-eabi* | ppc-*-sysv4*)		fmt=elf ;;
+  ppc-*-aeabi*)				fmt=elf ;;
   ppc-*-linux-*)			fmt=elf em=linux ;;
   ppc-*-solaris*)			fmt=elf em=solaris ;;
   ppc-*-rtems*)				fmt=elf ;;
diff -Naur binutils-2.25/ld/configure.tgt binutils-2.25_new/ld/configure.tgt
--- binutils-2.25/ld/configure.tgt	2014-10-14 02:32:04.000000000 -0500
+++ binutils-2.25_new/ld/configure.tgt	2015-02-10 05:25:14.293951859 -0600
@@ -556,7 +556,7 @@
 powerpc-*-vxworks*)
 			targ_emul=elf32ppcvxworks
 			targ_extra_emuls="elf32ppc elf32ppclinux elf32ppcsim" ;;
-powerpc*-*-elf* | powerpc*-*-eabi* | powerpc*-*-sysv* \
+powerpc*-*-elf* | powerpc*-*-eabi* | powerpc*-*-aeabi* | powerpc*-*-sysv* \
   | powerpc*-*-linux* | powerpc*-*-netbsd* | powerpc*-*-openbsd* \
   | powerpc*-*-solaris* | powerpc*-*-kaos* | powerpc*-*-vxworks*)
 			case "${targ}" in
