diff -Naur binutils-2.25/gas/testsuite/gas/ppc/e5500_dcbt.d binutils-2.25_new/gas/testsuite/gas/ppc/e5500_dcbt.d
--- binutils-2.25/gas/testsuite/gas/ppc/e5500_dcbt.d	1969-12-31 18:00:00.000000000 -0600
+++ binutils-2.25_new/gas/testsuite/gas/ppc/e5500_dcbt.d	2015-02-10 22:39:48.675952006 -0600
@@ -0,0 +1,11 @@
+#as: -mppc -me5500
+#objdump: -dr -Me5500
+#name: Power E5500 dcbt tests
+
+.*: +file format elf(32)?(64)?-powerpc.*
+
+Disassembly of section \.text:
+
+0+00 <start>:
+   0:	7c 00 2a 2c 	dcbt    0,r5
+   4:	7c 00 29 ec 	dcbtst  0,r5
diff -Naur binutils-2.25/gas/testsuite/gas/ppc/e5500_dcbt.s binutils-2.25_new/gas/testsuite/gas/ppc/e5500_dcbt.s
--- binutils-2.25/gas/testsuite/gas/ppc/e5500_dcbt.s	1969-12-31 18:00:00.000000000 -0600
+++ binutils-2.25_new/gas/testsuite/gas/ppc/e5500_dcbt.s	2015-02-10 22:39:48.675952006 -0600
@@ -0,0 +1,5 @@
+# Power E5500 dcbt tests
+	.section ".text"
+start:
+	dcbt	0, 0, 5
+	dcbtst	0, 0, 5
diff -Naur binutils-2.25/gas/testsuite/gas/ppc/e6500_dcbt.d binutils-2.25_new/gas/testsuite/gas/ppc/e6500_dcbt.d
--- binutils-2.25/gas/testsuite/gas/ppc/e6500_dcbt.d	1969-12-31 18:00:00.000000000 -0600
+++ binutils-2.25_new/gas/testsuite/gas/ppc/e6500_dcbt.d	2015-02-10 22:39:48.676952006 -0600
@@ -0,0 +1,11 @@
+#as: -mppc -me6500
+#objdump: -dr -Me6500
+#name: Power E6500 dcbt tests
+
+.*: +file format elf(32)?(64)?-powerpc.*
+
+Disassembly of section \.text:
+
+0+00 <start>:
+   0:	7c 00 2a 2c 	dcbt    0,r5
+   4:	7c 00 29 ec 	dcbtst  0,r5
diff -Naur binutils-2.25/gas/testsuite/gas/ppc/e6500_dcbt.s binutils-2.25_new/gas/testsuite/gas/ppc/e6500_dcbt.s
--- binutils-2.25/gas/testsuite/gas/ppc/e6500_dcbt.s	1969-12-31 18:00:00.000000000 -0600
+++ binutils-2.25_new/gas/testsuite/gas/ppc/e6500_dcbt.s	2015-02-10 22:39:48.676952006 -0600
@@ -0,0 +1,6 @@
+# Power E6500 dcbt tests
+        .section ".text"
+start:
+        dcbt    0, 0, 5
+        dcbtst  0, 0, 5
+
diff -Naur binutils-2.25/gas/testsuite/gas/ppc/ppc.exp binutils-2.25_new/gas/testsuite/gas/ppc/ppc.exp
--- binutils-2.25/gas/testsuite/gas/ppc/ppc.exp	2014-10-14 02:32:03.000000000 -0500
+++ binutils-2.25_new/gas/testsuite/gas/ppc/ppc.exp	2015-02-10 22:57:03.477952006 -0600
@@ -78,7 +78,9 @@
 	run_dump_test "e6500"
 	run_dump_test "e500mc64_nop"
 	run_dump_test "e5500_nop"
+	run_dump_test "e5500_dcbt"
 	run_dump_test "e6500_nop"
+	run_dump_test "e6500_dcbt"
 	run_dump_test "cell"
 	run_dump_test "power4_32"
 	run_dump_test "power6"
diff -Naur binutils-2.25/include/opcode/ppc.h binutils-2.25_new/include/opcode/ppc.h
--- binutils-2.25/include/opcode/ppc.h	2014-10-14 02:32:04.000000000 -0500
+++ binutils-2.25_new/include/opcode/ppc.h	2015-02-10 22:59:11.476952006 -0600
@@ -190,6 +190,9 @@
 /* Opcode is only supported by Power8 architecture.  */
 #define PPC_OPCODE_POWER8     0x2000000000ull
 
+/* Opcode is supported by Power E5500 */
+#define PPC_OPCODE_E5500      0x4000000000ull
+
 /* Opcode which is supported by the Hardware Transactional Memory extension.  */
 /* Currently, this is the same as the POWER8 mask.  If another cpu comes out
    that isn't a superset of POWER8, we can define this to its own mask.  */
diff -Naur binutils-2.25/opcodes/ppc-dis.c binutils-2.25_new/opcodes/ppc-dis.c
--- binutils-2.25/opcodes/ppc-dis.c	2014-10-14 02:32:04.000000000 -0500
+++ binutils-2.25_new/opcodes/ppc-dis.c	2015-02-10 23:04:45.005952006 -0600
@@ -117,8 +117,8 @@
     0 },
   { "e5500",    (PPC_OPCODE_PPC | PPC_OPCODE_BOOKE | PPC_OPCODE_ISEL
 		| PPC_OPCODE_PMR | PPC_OPCODE_CACHELCK | PPC_OPCODE_RFMCI
-		| PPC_OPCODE_E500MC | PPC_OPCODE_64 | PPC_OPCODE_POWER4
-		| PPC_OPCODE_POWER5 | PPC_OPCODE_POWER6
+		| PPC_OPCODE_E500MC | PPC_OPCODE_64 | PPC_OPCODE_E5500
+		| PPC_OPCODE_POWER4 | PPC_OPCODE_POWER5 | PPC_OPCODE_POWER6
 		| PPC_OPCODE_POWER7),
     0 },
   { "e6500",   (PPC_OPCODE_PPC | PPC_OPCODE_BOOKE | PPC_OPCODE_ISEL
diff -Naur binutils-2.25/opcodes/ppc-opc.c binutils-2.25_new/opcodes/ppc-opc.c
--- binutils-2.25/opcodes/ppc-opc.c	2014-12-23 02:47:10.000000000 -0600
+++ binutils-2.25_new/opcodes/ppc-opc.c	2015-02-10 23:45:51.734952006 -0600
@@ -2755,6 +2755,7 @@
 #define E500	PPC_OPCODE_E500
 #define E6500	PPC_OPCODE_E6500
 #define PPCVLE  PPC_OPCODE_VLE
+#define E5500  PPC_OPCODE_E5500
 #define PPCHTM  PPC_OPCODE_HTM
 
 /* The opcode table.
@@ -4711,7 +4712,8 @@
 {"mtvsrwz",	X(31,243),	XX1RB_MASK,   PPCVSX2,	PPCNONE,	{XT6, RA}},
 
 {"dcbtstt",	XRT(31,246,0x10), XRT_MASK,  POWER7,	PPCNONE,	{RA0, RB}},
-{"dcbtst",	X(31,246),	X_MASK,      POWER4,	PPCNONE,	{RA0, RB, CT}},
+{"dcbtst",	X(31,246),	X_MASK,      POWER4,	E5500|E6500,	{RA0, RB, CT}},
+{"dcbtst",	X(31,246),	X_MASK,      E5500|E6500, PPCNONE,	{CT, RA0, RB}},
 {"dcbtst",	X(31,246),	X_MASK,      PPC|PPCVLE, POWER4,	{CT, RA0, RB}},
  
 {"stbux",	X(31,247),	X_MASK,	     COM|PPCVLE, PPCNONE,	{RS, RAS, RB}},
@@ -4753,7 +4755,8 @@
 {"lscbx.",	XRC(31,277,1),	X_MASK,      M601,	PPCNONE,	{RT, RA, RB}},
 
 {"dcbtt",	XRT(31,278,0x10), XRT_MASK,  POWER7,	PPCNONE,	{RA0, RB}},
-{"dcbt",	X(31,278),	X_MASK,      POWER4,	PPCNONE,	{RA0, RB, CT}},
+{"dcbt",	X(31,278),	X_MASK,      POWER4,	E5500|E6500,	{RA0, RB, CT}},
+{"dcbt",	X(31,278),	X_MASK,      E5500|E6500, PPCNONE,	{CT, RA0, RB}},
 {"dcbt",	X(31,278),	X_MASK,      PPC|PPCVLE, POWER4,	{CT, RA0, RB}},
  
 {"lhzx",	X(31,279),	X_MASK,      COM|PPCVLE, PPCNONE,	{RT, RA0, RB}},
