diff -rNu gitold/locale/programs/locarchive.c git/locale/programs/locarchive.c
--- gitold/locale/programs/locarchive.c	2018-04-12 17:17:18.139309542 +0800
+++ git/locale/programs/locarchive.c	2018-04-13 11:16:07.066793890 +0800
@@ -568,11 +568,14 @@
 
   /* If ah has a non-NULL fname open that otherwise open the default.  */
   if (archivefname == NULL)
-    {
-      archivefname = default_fname;
-      if (output_prefix)
-        memcpy (default_fname, output_prefix, prefix_len);
-      strcpy (default_fname + prefix_len, ARCHIVE_NAME);
+    {         
+      archivefname = getenv("LOCALEARCHIVE");
+      if (archivefname == NULL) {
+              archivefname = default_fname;
+              if (output_prefix)
+                memcpy (default_fname, output_prefix, prefix_len);
+              strcpy (default_fname + prefix_len, ARCHIVE_NAME);
+      }
     }
 
   while (1)
@@ -585,7 +588,7 @@
 	     the default locale archive we ignore the failure and
 	     list an empty archive, otherwise we print an error
 	     and exit.  */
-	  if (errno == ENOENT && archivefname == default_fname)
+	  if (errno == ENOENT)
 	    {
 	      if (readonly)
 		{
