From 0def325c522c6c0df39562addab67e4fd150cb48 Mon Sep 17 00:00:00 2001
From: Chunrong Guo <chunrong.guo@nxp.com>
Date: Tue, 24 Apr 2018 16:25:57 +0800
Subject: [PATCH] fix install error

Signed-off-by: Chunrong Guo <chunrong.guo@nxp.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index d40c839..32c0abe 100644
--- a/Makefile
+++ b/Makefile
@@ -9,6 +9,6 @@ INSTALL ?= install
 ARCH ?= arm64
 
 install:
-	$(INSTALL) -d --mode 755 $(INSTALL_DIR)/usr/local/bin
-	sudo cp -r ${ARCH}/bootstrap-enroll $(INSTALL_DIR)/usr/local/bin/
+	$(INSTALL) -d --mode 755 $(INSTALL_DIR)/usr/bin
+	cp -r ${ARCH}/bootstrap-enroll $(INSTALL_DIR)/usr/bin/
 
-- 
2.7.4

